frontend/package.json

```diff
@@ -12,8 +12,8 @@
     "prebuild": "yarn copy-config",
     "test": "react-scripts test",
     "eject": "react-scripts eject",
-    "lint": "eslint --ext js src",
-    "lint:fix": "eslint --fix --ext js src"
+    "lint": "node_modules/.bin/eslint --ext js src",
```

Comment:
I can only guess, since I haven't tested it without the prefix: 
I assume that `eslint` is not in the PATH. It was added to path in mod-main by one of those:
```
    "eslint-config-prettier": "^8.8.0",
    "eslint-plugin-import": "^2.27.5",
    "eslint-import-resolver-alias": "^1.1.2",
    "eslint-plugin-prettier": "^4.2.1",
```
Though, it's just an assumption.